doc/rst/developer/bestPractices/Deprecation.rst

```diff
@@ -16,7 +18,7 @@ This is the deprecation process:
 
 The deprecation message uses the following format:
 
-..code-block::
+.. code-block::
 
     foo is deprecated - please use bar
```

Comment:
This code block accidentally includes the paragraph after it as well.  I think you need to de-dent it?